globus_cli/config.py

```diff
@@ -94,6 +110,30 @@ def get_transfer_refresh_token():
     return lookup_option(TRANSFER_RT_OPTNAME)
 
 
+def get_auth_access_token():
```

Comment:
That's a good point. I suppose there's no reason to have separate methods for getting the tokens from the config...

I'll consolidate them into a single method.